datanode/api/trading_data_v2.go

```diff
@@ -907,7 +907,11 @@ func (t *tradingDataServiceV2) GetERC20ListAssetBundle(ctx context.Context, req
 	}
 
 	// now we pack them
-	pack := "0x"
+	pack := ""
+	if len(signatures) > 0 {
```

Comment:
Wondering if in the case when there are no signatures, then we should treat it as not-found (or null or whatever the conclusion was to return in not found cases)? Because the query is "can I have the ListAssetBundle for this asset please" and if there are no signatures, then it means no bundle has been sent out from core.

Reply:
I really don't have any strong feelings eitherway.

For `GetERC20MultiSigSignerAddedBundles` and `GetERC20MultiSigSignerRemovedBundles` you are right, they should probably be `ListXXX` because it does return multiple bundles, and in this case if there is an `GetAddedEvent()` then the signatures will (should) follow, but could still just be omitted from the list until they are through.

For `GetERC20ListAssetBundle` its a little different because the asset could exist, have been rejected at the proposal stage and so no signatures will come through (same with `GetERC20WithdrawalApproval`, a withdrawal can be rejected). I guess I was just thinking it might give false hope if the API returns _something_ even though the bit they really need from it is never coming.

Maybe @vegaprotocol/frontend or @edd have an opinion?